ckanext/bpatheme/static/bpatheme.css

```diff
@@ -236,7 +236,25 @@ a, a:active, a:hover, a:focus {
   border:none;
 }
 
-
+/* fix tablet view */
+@media (max-width: 768px){
+  .wrapper {
+      margin: 0 0;
+  }
+}
+
+/* Make sidebar wider to display large tags*/
+@media (min-width: 980px){
+  .wrapper {
+      background-position: 10% 0px;
+  }
+  .span3 {
+    width: 31%;
+  }
+  .span9 {
+    width: 67%;
+  }
+}
```

Comment:
I think it would be better to scope these changes - create new CSS classes and apply where needed, rather than modifying the meaning of the core bootstrap span3 and span9 classes.